example/src/creditCards.ts

```diff
@@ -0,0 +1,28 @@
+import dotenv from 'dotenv'
+import Pluggy from 'pluggy-js'
+import { sleep, PLUGGY_BANK_CREDENTIALS, PLUGGY_BANK_CONNECTOR } from './utils'
+
+dotenv.config()
+
+const { PLUGGY_API_KEY: pluggyApiKey, PLUGGY_API_URL } = process.env
+const baseUrl = PLUGGY_API_URL || undefined
+const baseClient = new Pluggy(pluggyApiKey, baseUrl)
+
+void (async function(): Promise<void> {
+  const { accessToken: connectToken } = await baseClient.createConnectToken()
+  const client = new Pluggy(connectToken, baseUrl)
```

Comment:
It would be great to pass the baseUrl by env variable instead of passing it explicitly.

Reply:
It's being retrieved from an env var above

Reply:
Yep, i mean to have it internal in the client, rather than expose a constructor parameter.

Reply:
So we should rework pluggy-js to retrieve the prop from an env var right? for now I've removed this setting from this example PR, I think I can improve the configurability in another PR to the main module